src/ahtml-node.adb

```diff
@@ -141,4 +141,4 @@ package body AHTML.Node is
        Attrs => Attrs_Vec.Empty_Vector,
        Children => Index_Vec.Empty_Vector));
 
-end AHTML.Node;
```

Comment:
Noticed you've gotten rid of the newline. Could you amend this and force push? Thanks!

Reply:
hi! i use svn2git and force push doesn't &nbspwork there

Reply:
fascinating